content/master/concepts/composition.md

```diff
@@ -965,7 +965,7 @@ fallback to the input value by setting the `fallbackTo` field to `Input`.
 
 `string`. Transforms string values. 
 * string transform type `Format`, Currently only Go style fmt is supported. [Go style `fmt`][pkg/fmt] is supported.
-* string transform type `Convert`, accepts one of `ToUpper`, `ToLower`, `ToBase64`, `FromBase64`.
+* string transform type `Convert`, accepts one of `ToUpper`, `ToLower`, `ToBase64`, `FromBase64`, `ToJson`.
```

Comment:
```suggestion
* string transform type `Convert`, accepts one of `ToUpper`, `ToLower`, `ToBase64`, `FromBase64`, `ToJson`. `ToSha1`, `ToSha256`, `ToSha512`.
```

Reply:
done in: https://github.com/crossplane/docs/pull/406/commits/8ae216c435601a8d02aa723868992c91b0427efa